### PR TITLE
terminus-font: update to version 4.48

### PR DESCRIPTION
**Pull request by exg**

### CODE REVIEW

File: x11/terminus-font/Portfile [hunk 1]
```diff
@@ -18,8 +17,8 @@ platforms       darwin
 homepage        http://terminus-font.sourceforge.net/
 master_sites    sourceforge:project/terminus-font/terminus-font-${version}
 
-checksums       rmd160  51dad30b36b9ef2fc20e6daa70df41507f082da2 \
-                sha256  0f1b205888e4e26a94878f746b8566a65c3e3742b33cf9a4e6517646d5651297
+checksums       rmd160  07855292c30db918fb9952a061cb746d9eb1e3da \
+                sha256  34799c8dd5cec7db8016b4a615820dfb43b395575afbb24fc17ee19c869c94af
```
Comment (mf2k) [inline x11/terminus-font/Portfile]: We are now adding the `size` parameter to `checksums`.

File: x11/terminus-font/Portfile [hunk 2]
```diff
@@ -18,8 +17,8 @@ platforms       darwin
...
 
 depends_build   port:python36 \
```
Comment (mf2k) [inline x11/terminus-font/Portfile]: Can this be changed to `python37`?

File: x11/terminus-font/Portfile (hunk 1 above)
Comment (exg) [reply x11/terminus-font/Portfile]: Thanks,I wasn't aware of that. Added.

File: x11/terminus-font/Portfile (hunk 2 above)
Comment (exg) [reply x11/terminus-font/Portfile]: Yes, changed.

**Comment by mf2k** [comment]
Thank you! I hate to nitpick, but please add the fact that we are switching to python 3.7 in the commit message.

**Pull request merged**